tools/generate-integration-docs-screenshot

```diff
@@ -41,6 +41,7 @@ from zerver.lib.actions import (
 from zerver.lib.integrations import (
     DOC_SCREENSHOT_CONFIG,
     INTEGRATIONS,
+    Integration,
```

Comment:
Maybe add a commit before this to rename `INTEGRATIONS` to `INTEGRATIONS_INFO`  for better readability.

Reply:
Hmm, not sure I agree with that renaming; what's your thinking?  We have related settings like WEBHOOK_INTEGRATIONS.